### ISSUE TITLE
Tooltip doesn't fit the content

### ISSUE THREAD

**Issue by Kearsan** [atomiks/tippyjs#446]
Hello!

Thanks a lot for this amazing library, you really do a great job.

My issue is about dynamic content loaded with ajax. On specific tooltips I load HTML content (sent via ajax). I've an extra space around the content inside the tooltip, but I want the tooltip to fit perfectly with the content inside it. My content is a block with specified width and height, is there a way to avoid this extra space around the content ?

Thanks!

**Comment by atomiks** [comment]
Hi,

A CodePen demo would really help to let me fiddle with the styles and see what's happening 🙂

**Comment by atomiks** [comment]
Actually I'm pretty sure I know what you're talking about after re-reading that. It's probably the `padding`.

You can specify a custom theme and remove it:

```js
tippy(ref, {
  theme: 'ajax'
})
```

```css
.tippy-tooltip.ajax-theme {
  padding: 0;
}
```

**Comment by Kearsan** [comment]
Thanks for your quick answer. Indeed the issue is about padding, that's what I thought first but doesn't work on my side. So I just made a codepen to show you my issue and... your solution is working just fine on codepen! So it seems that the mistake is on my side, maybe because of my content, I'll take a deeper look at it and come back to you if I can't figured it out.

Thanks again!

**Comment by Kearsan** [comment]
I figured that a css rule on my side was breaking the content, so everything is ok now. Thanks for putting me on a good lead!
